keps/sig-node/4742-node-topology-downward-api/README.md

```diff
@@ -0,0 +1,411 @@
+that supports copying topology labels to Pods.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+A feature gate `PodTopologyLabelsAdmission` will be added to the kube-apiserver to enable or disable this feature.
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: `PodTopologyLabelsAdmission`
+  - Components depending on the feature gate: `kube-apiserver`
+- [ ] Other
+  - Describe the mechanism:
+  - Will enabling / disabling the feature require downtime of the control
+    plane?
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node?
+
+###### Does enabling the feature change any default behavior?
+
+Yes, all Pods will have topology labels copied over.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes it can, but Pods created while the feature is enabled will continue to have topology labels.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+New Pods will have topology labels copied over. Older Pods created while the feature was disabled will be missing topology labels.
+
+###### Are there any tests for feature enablement/disablement?
+
+Tests will be added to ensure feature gate works as expected.
+
+### Rollout, Upgrade and Rollback Planning
+
+TODO for Beta.
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Failure on rollout is unlikely since we do not override any existing topolgoy labels on Pods.
+Already running workloads that is already copying topology labels to Pods should be untouched by the new admission plugin.
+
+###### What specific metrics should inform a rollback?
+
+N/A
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+Upgrade / rollback will be manually tested before graduation to Beta.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No.
+
+### Monitoring Requirements
+
+TODO for GA.
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Look for topology labels in Pod labels.
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [ ] Events
+  - Event Reason:
+- [X] API .status
+  - Condition name:
+  - Other field: check Pod labels
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+N/A
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [X] Other (treat as last resort)
```

Comment:
See above - we want to make sure bindings aren't failing, hopefully there is some metric for that (even if it's higher level scheduling metrics)

Reply:
added a scheduler metric here